Assets/MixedRealityToolkit.Staging/UnityAR/UnityARCameraSettings.cs

```diff
@@ -6,11 +6,11 @@
 using Microsoft.MixedReality.Toolkit.Utilities;
 using UnityEngine;
 
-#if UNITY_IOS || UNITY_ANDROID
+#if (UNITY_IOS || UNITY_ANDROID) && ARFOUNDATION_PRESENT
```

Comment:
Do we want to move `ARFOUNDATION_PRESENT` into the asmdef constraints now?

Reply:
We may want to, indeed.

Reply:
One thing i would like to confirm when trying this idea is that profiles do not break.

Reply:
I've been thinking about this with some of the XR SDK work, and I think (if I'm correctly interpreting "break" here) that if a type or data provider is referenced in a profile but its assembly won't load in the current context, then you just shouldn't have that type referenced. Either fix the issue (add ARFoundation, etc) or remove the type, because it won't work.

Reply:
I went ahead and made this change, but do let me know if you have concerns or a scenario to test and I'll make sure things are good!

Reply:
This change breaks the ability to reference the assembly in non-AR Foundation platforms (ex: standalone and UWP)... Looks like we need a different solution.

Reply:
![image](https://user-images.githubusercontent.com/13281406/71693218-baa97700-2d60-11ea-8b4d-8eb2c260119b.png)

Reply:
I'm not sure I think that's a bad thing. It's at least a visual indicator that the current build settings don't support the type, then the supported platforms list prevents it from being referenced at runtime.

Reply:
But I'm definitely down for a different solution either way